core/common/src/annotation/TextAnnotation.ts

```diff
@@ -65,6 +65,45 @@ export interface TextFrameStyleProps {
   borderWeight?: number;
 }
 
+/**
+ * Specifies where to attach a leader line to a [[TextAnnotation]].
+ * The leader line can be attached to a key point, to the first or last line of text, or to the nearest point on the text block frame.
+ * @beta
+ */
+export type LeaderAttachmentMode =
```

Comment:
```suggestion
export type LeaderAttachment =
```

This type **has** a `mode`, it is not itself a mode.